bindata/scripts/kargs.sh

```diff
@@ -7,49 +7,54 @@ declare -a kargs=( "$@" )
 ret=0
 args=$(chroot /host/ cat /proc/cmdline)
 
-if chroot /host/ test -f /run/ostree-booted ; then
-    for t in "${kargs[@]}";do
-        if [[ $command == "add" ]];then
-          if [[ $args != *${t}* ]];then
-              if chroot /host/ rpm-ostree kargs | grep -vq ${t}; then
-                  chroot /host/ rpm-ostree kargs --append ${t} > /dev/null 2>&1
+IS_OS_UBUNTU=true; [[ "$(chroot /host/ grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
+
+# Kernerl args comfiguration isn't supported for Ubuntu now, so we shouldn't do anything here
+if ! ${IS_OS_UBUNTU} ; then
```

Comment:
Hi let's make this one simpler, please.

can you do something like

```
if ${IS_OS_UBUNTU} ; then
  echo 0
  exit 0
fi
```

Reply:
done